components/datasciencepipelines/datasciencepipelines.go

```diff
@@ -92,6 +95,35 @@ func (d *DataSciencePipelines) GetComponentName() string {
 	return ComponentName
 }
 
+func (d *DataSciencePipelines) GetUpstreamReleaseStatus() status.UpstreamReleases {
+	var componentVersion semver.Version
+	var repositoryURL string
+	var releaseDetails = make([]status.ComponentReleaseStatus, 0)
+	var releaseStatus status.UpstreamReleases
+
+	env, err := godotenv.Read(filepath.Join(deploy.DefaultManifestPath, ComponentName, ".env"))
```

Comment:
We have similar functionality related to ApplyParams().Do we need to bring another dependency?

Reply:
i am not fond of introducing godotenv. but does not feel like whats in ApplyParams can be easiliy work as "read 
 file+convert to map"

Reply:
i leave this to @ykaliuta for more comments

Reply:
> i am not fond of introducing godotenv. but does not feel like whats in ApplyParams can be easiliy work as "read file+convert to map"

doesn't parseParams() do exactly that?

Reply:
So, the file as it is now actually in a custom format with spaces around `=` and `"` around values. Not what I would expect from `env`, but go has own opinion :)

Reply:
i am on the side of using key=value, not to have wider support for both key=value and key:value
which i think godotenv supporr the later case.
but since i do not see any detail in the refinement doc, except we need env file, so i do not know what has been agreed with all stakeholders, thats why i think the parseParams() only work with "="  and if " is needed.

Reply:
@ykaliuta even with strict format (KEY=VALUE), it get this error with parseParams() `Invalid character(s) found in major number "\"1"`

Reply:
> @ykaliuta even with strict format (KEY=VALUE), it get this error with parseParams() `Invalid character(s) found in major number "\"1"`

probably, you still use `"` which in _strict_ format becomes part of the value.

Reply:
@ykaliuta yes my bad, after removing quotes now it is working